docs/HowToFAQ/FAQTypeParameterRestriction.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Is it possible to restrict a type parameter to be a reference type? I see you can use T(!new) but I’m looking for the opposite.
+---
+
+## Question
+
+Is it possible to restrict a type parameter to be a reference type? I see you can use T(!new) but I’m looking for the opposite.
```

Comment:
Write "T(!new)" as "`T(!new)`".

Reply:
Done (and elsewhere in the answer-- though formatting is not allowed in the title)